velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -529,6 +543,10 @@ TEST_F(StringImplTest, pad) {
   runTestUserError("text", -1, "a");
   runTestUserError(
       "text", ((int64_t)std::numeric_limits<int32_t>::max()) + 1, "a");
+  // Additional tests with bad unicode bytes.
+  runTest("abcd\uFFFFef", 6, "0", "abcd\uFFFFe", "abcd\uFFFFe");
+  runTest("abcd\uFFFFef", 9, "0", "00abcd\uFFFFef", "abcd\uFFFFef00");
```

Comment:
SELECT lpad(concat('abcd', FROM_UTF8(x'FF FF'), 'ef'), 9, '0');
gives me :
0abcd��ef
and
abcd��ef0
 

<br class="Apple-interchange-newline">

Reply:
As noted \uFFFF not = \xFF \xFF